src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -59,27 +59,33 @@ private static object RegExSetup(PowerFxConfig config)
             return null;
         }
 
-        private static PowerFxConfig AllEnumsSetup(PowerFxConfig config)
+        private static PowerFxConfig AllEnumsSetup(PowerFxConfig config)
         {
-            return PowerFxConfig.BuildWithEnumStore(new EnumStoreBuilder().WithDefaultEnums(), new TexlFunctionSet(), config.Features);
-        }
-
-        private static object OptionSetTestSetup1(PowerFxConfig config)
+            return PowerFxConfig.BuildWithEnumStore(new EnumStoreBuilder().WithDefaultEnums(), new TexlFunctionSet(), config.Features);
+        }
+
+        private static object OptionSetTestSetup1(PowerFxConfig config)
         {
-            OptionSet optionSet = new OptionSet("OptionSet", DisplayNameUtility.MakeUnique(new Dictionary<string, string>()
-            {
-                    { "option_1", "Option1" },
-                    { "option_2", "Option2" }
+            OptionSet optionSet = new OptionSet("OptionSet", DisplayNameUtility.MakeUnique(new Dictionary<string, string>()
+            {
+                    { "option_1", "Option1" },
+                    { "option_2", "Option2" },
+                    { "option-3", "Option-3" },
+                    { "Is", "Option IS" }, // Reserved word
+                    { "Self", "Option SELF" }, // Keyword
+                    { "is", "Option is low case" }, // Not a reserved word
+                    { "Self", "Option self low case" }, // Not a keyword

```

Comment:
>Self [](http://example.com/codeflow?start=23&length=4)

I don't think we can have two keys with the same name. Did you mean to call it `self`?

Reply:
Yes. Thanks for catching this.